### ISSUE TITLE
removing hidden files 

### ISSUE THREAD

**Issue by rfraimow** [WGBH/preservation-workflow#44]
Currently, the script generates metadata for hidden files and folders, then deletes them out of the final result, taking up extra computing and processing time. We would like to provide an option in the script that cleans hidden files and folders out of the drive or folder being preserved before any preservation activities (copy, diff, FITS) occur.

**Comment by sroosa** [comment]
This should be charged to General MLA - Non-overhead 156000. I can't seem to add labels in this repo, or I would just create a label to say this.

**Comment by jasoncorum** [comment]
@rfraimow - forgot to tell you yesterday that they way to use the new flag is to enter "no-hidden" before the path to the source:
```
$ ruby preserve.rb no-hidden /path/to/SOURCE /path/to/METADATA /path/to/DESTINATION1 /path/to/DESTINATION2
```
if you want hidden files just use the regular command:
```
$ ruby preserve.rb /path/to/SOURCE /path/to/METADATA /path/to/DESTINATION1 /path/to/DESTINATION2
```

**Comment by afred** [comment]
closed by #45
